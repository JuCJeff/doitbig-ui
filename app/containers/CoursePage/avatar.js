import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import Avatar from '@material-ui/core/Avatar';
import Grid from '@material-ui/core/Grid';
import Image from '../../images/Jeff.jpg';

const styles = {
  avatar: {
    margin: 10,
  },
  bigAvatar: {
    margin: 10,
    width: 60,
    height: 60,
  },
};

function ImageAvatars(props) {
  const { classes } = props;
  return (
    <Grid>
      <Avatar alt="Remy Sharp" src={Image} className={classes.avatar} />
      {/* <Avatar alt="Remy Sharp" src="/static/images/avatar/1.jpg" className={classes.bigAvatar} /> */}
    </Grid>
  );
}

ImageAvatars.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(ImageAvatars);
