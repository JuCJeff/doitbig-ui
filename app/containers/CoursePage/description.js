import React from 'react';
import PropTypes from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import Paper from '@material-ui/core/Paper';
import Typography from '@material-ui/core/Typography';
import Image from '../../images/STS.jpg';

const styles = theme => ({
  root: {
    ...theme.mixins.gutters(),
    paddingTop: theme.spacing.unit * 2,
    paddingBottom: theme.spacing.unit * 2,
  },
});

function PaperSheet(props) {
  const { classes } = props;

  return (
    <div>
      <Paper className={classes.root} elevation={1}>
        <Typography variant="h5" component="h3">
          This is a Wordpress Course.
        </Typography>
        <img src={Image} className="images" alt="banner" />
        <Typography component="p">
          WordPress.com offers a simple solution for getting basic websites up
          and running quickly, with no coding or technical knowledge needed.
        </Typography>
        <Typography component="p">
          This course provides an introduction to WordPress.com and guides you
          through the process of setting up a simple site.
        </Typography>
      </Paper>
    </div>
  );
}

PaperSheet.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(PaperSheet);
