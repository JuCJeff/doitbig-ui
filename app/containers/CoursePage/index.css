.images {
    display: block;
    margin-left: auto;
    margin-right: auto;
    width: 100%;
}

.description {
    width: 50%;
}