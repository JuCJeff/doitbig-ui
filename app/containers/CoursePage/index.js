import React from 'react';
import Description from './description';
import Avatar from './avatar';
import NavigationBar from '../../components/NavigationBar/loadable';
import Image from '../../images/STS.jpg';
import Footer from '../../components/Footer';
import './index.css';

/* eslint-disable react/prefer-stateless-function */
export default class CoursePage extends React.PureComponent {
  render() {
    return (
      <div>
        <div>
          <NavigationBar />
        </div>
        <img src={Image} className="images" alt="banner" />
        <div className="description">
          <Description />
        </div>
        <Avatar />
        <div>
          <Footer />
        </div>
      </div>
    );
  }
}
