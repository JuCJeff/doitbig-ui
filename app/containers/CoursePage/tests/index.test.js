import React from 'react';
import { shallow } from 'enzyme';

import CoursePage from '../index';

describe('<CoursePage />', () => {
  it('should render the page message', () => {
    const renderedComponent = shallow(
      <CoursePage location={{ search: '12345' }} />,
    );
    expect(renderedComponent.contains(<p>Loading...</p>)).toEqual(true);
  });
});
