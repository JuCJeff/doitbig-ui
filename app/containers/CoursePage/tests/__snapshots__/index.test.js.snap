// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoursePage /> should render the page message 1`] = `
<div>
  <div>
    <LoadableComponent />
  </div>
  <img
    alt="banner"
    className="images"
    src="IMAGE_MOCK"
  />
  <div
    className="description"
  >
    <WithStyles(PaperSheet) />
  </div>
  <WithStyles(ImageAvatars) />
</div>
`;
